compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -1014,7 +1014,8 @@ public BLangNode transform(ObjectFieldNode objFieldNode) {
         Optional<Node> doc = getDocumentationString(objFieldNode.metadata());
         simpleVar.markdownDocumentationAttachment = createMarkdownDocumentationAttachment(doc);
 
-        addRedonlyQualifier(objFieldNode.readonlyKeyword(), objFieldNode.typeName(), simpleVar);
+        // TODO: add changes for `final`
+//        addRedonlyQualifier(objFieldNode.readonlyKeyword(), objFieldNode.typeName(), simpleVar);
```

Comment:
Do we need this commented line?

Reply:
there's going to be another PR by @MaryamZi  on top of this. Changes needed beyond parser is addressed there. If parser changes are ok, we can close this PR and directly merge that PR also.

Reply:
Yeah, +1 to merge them with my PR to avoid merging anything in an inconsistent state.